lib/controller/index.js

```diff
@@ -110,12 +114,15 @@ let Controller = BaseController;
 Controller = require('./mixins/resolve-path')(Controller);
 Controller = require('./mixins/base-url')(Controller);
 Controller = require('./mixins/translate')(Controller);
+Controller = require('./mixins/journey-model')(Controller);
 Controller = require('./mixins/session-model')(Controller);
 Controller = require('./mixins/check-session')(Controller);
 Controller = require('./mixins/check-progress')(Controller);
 Controller = require('./mixins/csrf')(Controller);
 Controller = require('./mixins/invalidate-fields')(Controller);
 Controller = require('./mixins/back-links')(Controller);
+Controller = require('./mixins/next-step')(Controller);
+Controller = require('./mixins/edit-step')(Controller);
```

Comment:
I've added tests for these and added a looped test that checks the order these are mixed in